src/Compilers/CSharp/Test/Semantic/Semantics/GlobalUsingDirectiveTests.cs

```diff
@@ -5011,5 +5011,121 @@ class C2 {}
             Assert.Equal("alias1=C1", model.GetSpeculativeAliasInfo(tree.GetRoot().Span.End, alias1, SpeculativeBindingOption.BindAsExpression).ToTestDisplayString());
             Assert.Equal("alias1=C1", model.GetSpeculativeAliasInfo(tree.GetRoot().Span.End, alias1, SpeculativeBindingOption.BindAsTypeOrNamespace).ToTestDisplayString());
         }
+
+        [Fact]
+        public void GlobalAliasToType1()
+        {
+            var source1 = @"
+global using X = int;
+";
+            var source2 = @"
+class C
+{
+    X Goo() => default;
```

Comment:
nit: To verify the `X` is `int`, consider `X Goo(int i) => i;` instead.

Reply:
done.